examples/labels.py

```diff
@@ -20,56 +20,52 @@
 from config_loader import try_load_from_file
 
 config = {
-    "ip": "172.16.102.59",
+    "ip": "10.1.20.1",
```

Comment:
can you give dummy credentials rather than our dcs credentials?

Reply:
done